src/web/routes.rs

```diff
@@ -71,7 +71,7 @@ pub(super) fn build_routes() -> Routes {
         super::builds::build_list_handler,
     );
     routes.internal_page(
-        "/crate/:name/:version/source",
+        "/crate/:name/:version/source/",
```

Comment:
Why was this change made? This seems to prevent navigating to the slash-less version, e.g. https://docs.rs/crate/rcc/0.2.0/source now gives a 404.

Reply:
Maybe a backward-compatible redirect would be nice? IIRC I have experienced docs.rs breaking backward compatibility.

Reply:
Because the contents of the source page rely on a slash being at the end of the URL. I can add a redirect from slash-less to the URL with the slash.